docs/getting_started/installation.rst

```diff
@@ -136,6 +136,21 @@ install the assets:
     bin/console cache:clear
     bin/console assets:install
 
+Webpack Encore Integration
+--------------------------
+
+When the assets are installed add following to ``config/sonata_admin.yaml``
+
+.. code-block:: yaml
+
+    webpack_encore:
+        builds:
+            sonata_admin: '%kernel.project_dir%/public/bundles/sonataadmin/dist'
+            # Handling multiple entry points. For example app entry
+            #app: '%kernel.project_dir%/public/build'
```

Comment:
I am for removing the comments

Reply:
How do we explain to someone who already has webpack? than he needs to do that.

Reply:
why? he already has an entry with `app: ...` in another file and the get merged, right?

Reply:
Probably not because he uses the `assets` config section in `sonata_admin.yaml`